src/vos/vos_io.c

```diff
@@ -1496,7 +1497,7 @@ dkey_update(struct vos_io_context *ioc, uint32_t pm_ver, daos_key_t *dkey,
 	    uint16_t minor_epc)
 {
 	struct vos_object	*obj = ioc->ic_obj;
-	daos_handle_t		 ak_toh;
+	daos_handle_t		 ak_toh = DAOS_HDL_INVAL;
```

Comment:
Is this change and the next one related?  It doesn't obviously appear to be.

Reply:
Yes, this change from testing without memset(ptr, 0, size).